.gitignore

```diff
@@ -28,3 +28,4 @@ bin/
 /lineman/app/css/modules.scss
 .powenv
 /lineman/app/css/modules.scss
+config/local_env.yml
```

Comment:
We already use `dotenv` for this purpose (loading local env variables for dev/test), so I don't think we need another method.